flowmachine/flowmachine/core/server/query_schemas/flowmachine_query.py

```diff
@@ -49,36 +51,5 @@ class FlowmachineQuerySchema(OneOfSchema):
         "location_introversion": LocationIntroversionSchema,
         "total_network_objects": TotalNetworkObjectsSchema,
         "dfs_metric_total_amount": DFSTotalMetricAmountSchema,
+        "spatial_aggregate": SpatialAggregateSchema,
     }
-
-    def get_obj_type(self, obj):
```

Comment:
Kinda nice that this boilerplate can be removed as a consequence of removing the `query_params` property (because this is only needed by the call to `FlowmachineQuerySchema().dump()`. 👍